ext/greenmat/autolink.c

```diff
@@ -223,7 +223,7 @@ sd_autolink__email(
 		return 0;
 
 	for (link_end = 0; link_end < size; ++link_end) {
-		uint8_t c = data[link_end];
+		char c = data[link_end];
```

Comment:
I understand this change forces the `c` to be virtually 7-bit ASCII but it's a bit obscure. Maybe the way in https://github.com/vmg/redcarpet/pull/463 and defining a macro something like `is_ascii()` (`c < 0x7f`) would be more explicit.